test/sharing/gist_loader_test.dart

```diff
@@ -0,0 +1,605 @@
+import 'dart:convert';
+
+import 'package:dart_pad/sharing/gists.dart';
+import 'package:test/test.dart';
+import 'package:http/http.dart' as http;
+import 'package:http/testing.dart';
+
+void main() => defineTests();
+
+void defineTests() {
+  MockClient mockClient;
```

Comment:
👍

Reply:
So nice to work on something that can be easily isolated and tested for a change.